app/models/artist_public_data.rb

```diff
@@ -19,9 +19,32 @@ def path(context)
       if @artist.image.present?
         @artist.image.file.url
       end
+    else
+      "/artists/#{@artist.id}"
     end
   end
 
+  def birth_date
```

Comment:
If we take Born/Died off the artist page, we can also remove `#birth_date` and `#deceased_date`.